test/inductor/test_cuda_repro.py

```diff
@@ -1021,8 +1021,8 @@ def forward(self, x):
             enable_flash=True, enable_math=False, enable_mem_efficient=False
         ):
             input1 = torch.rand(5, 512, 1024, device="cuda", dtype=torch.float16)
-            input2 = torch.rand(5, 513, 1024, device="cuda", dtype=torch.float16)
-            input3 = torch.rand(5, 514, 1024, device="cuda", dtype=torch.float16)
+            input2 = torch.rand(5, 516, 1024, device="cuda", dtype=torch.float16)
+            input3 = torch.rand(5, 520, 1024, device="cuda", dtype=torch.float16)
```

Comment:
with my change, we end up triggering recompiles when size[0]*size[1] is not divisible by 20. not sure if this is an issue...

I guess it would be ideal to only add the divisibility guard if we know it unlocks a fusion, but I'm not sure how difficult that is to do